utils.js

```diff
@@ -801,14 +894,110 @@ HDSigner.prototype.signPSBT = async function (psbt) {
   return psbt
 }
 
+/* convertToTrezorFormat
+Purpose: Convert syscoin PSBT to Trezor format
+Param psbt: Required. Partially signed transaction object
+Returns: trezor params to signTransaction
+*/
+TrezorSigner.prototype.convertToTrezorFormat = function (psbt) {
+  const trezortx = {};
+
+  trezortx.coin = 'sys';
```

Comment:
cannot make this hardcoded, how do we make this dynamic? need to support other coins like btc, maybe pass in network params?